src/components/Wizard/NewEditImportPane.tsx

```diff
@@ -108,11 +118,71 @@ function NewForm({ onNewMessage }: NewFormProps) {
   );
 }
 
-function EditForm() {
+function EditForm({ onEditMessage, onDeleteMessage }: EditFormProps) {
+  const formContext = useForm<EditFormData>({ mode: "onChange" });
+  const savedMessages = JSON.parse(localStorage.savedMessages);
```

Comment:
This should come down as a prop because React won't know when localStorage changes so the component will not render with new messages.